test/e2e/cloud-provider-azure.go

```diff
@@ -61,6 +61,10 @@ func InstallCalicoAndCloudProviderAzureHelmChart(ctx context.Context, input clus
 		options.StringValues = append(options.StringValues, fmt.Sprintf("cloudNodeManager.imageTag=%s", os.Getenv("IMAGE_TAG_CNM")))
 	}
 
+	if input.ConfigCluster.Flavor == "flatcar" {
+		options.StringValues = append(options.StringValues, "cloudControllerManager.caCertDir=/usr/share/ca-certificates")
```

Comment:
do we need to add this anywhere else in docs or tilt @invidian?

Reply:
I am not familiar with CAPZ documentation regarding external CCM, let me have a look...

Reply:
I guess Tiltfile could include this logic and https://capz.sigs.k8s.io/topics/addons.html section about CCM could have it mentioned.

Reply:
> I guess Tiltfile could include this logic and https://capz.sigs.k8s.io/topics/addons.html section about CCM could have it mentioned.

Updated